ivy/functional/ivy/linear_algebra.py

```diff
@@ -714,7 +714,8 @@ def cholesky(
 
      3. Returns a lower-triangular Cholesky factor
 
-     >>> x = ivy.Container(a = ivy.array([[3., -1.], [-1., 3.]]) , b = ivy.array([[2., 1.], [1., 1.]]))
+     >>> x = ivy.Container(a = ivy.array([[3., -1.], [-1., 3.]]) ,
```

Comment:
you would need a `\` character at the end of the line for this to pass docstring tests